abel/direct.py

```diff
@@ -224,45 +222,3 @@ def is_uniform_sampling(r):
     ddr = np.diff(dr)
     return np.allclose(ddr, 0, atol=1e-13)
 
-
-def _abel_sym():
-    """
-    Analytical integration of the cell near the singular value
-    in the abel transform.
-    The resulting formula is implemented in abel.lib.direct.abel_integrate
-    """
-    from sympy import symbols, simplify, integrate, sqrt
-    from sympy.assumptions.assume import global_assumptions
-    r, y, r0, r1, r2, z, dr, c0, c_r, c_rr, c_z, c_zz, c_rz = symbols(
-            'r y r0 r1 r2 z dr c0 c_r c_rr c_z c_zz c_rz', positive=True)
-    f0, f1, f2 = symbols('f0 f1 f2')
-    global_assumptions.add(Q.is_true(r > y))
-    global_assumptions.add(Q.is_true(r1 > y))
-    global_assumptions.add(Q.is_true(r2 > y))
-    global_assumptions.add(Q.is_true(r2 > r1))
-    P = c0 + (r-y)*c_r  # + (r-r0)**2*c_rr
-    K_d = 1/sqrt(r**2-y**2)
-    res = integrate(P*K_d, (r, y, r1))
-    sres = simplify(res)
-    print(sres)
-
-
-def reflect_array(x, axis=1, kind='even'):
```

Comment:
I think we still have code depending on this function in my former lab, so I would rather keep it if possible to avoid breaking things.

Reply:
Perhaps, the function may be added to `abel/tools/symmetry.py` ?

Reply:
@stggh Sounds good to me.

Reply:
I made the move to `tools.symmetry.reflect_array`. But basically this is just a wrapper for np.flipud and np.fliplr?

If you really don't want to break the code in your previous lab, should we leave the function exactly where it is and include a depreciation warning?

Reply:
OK, I checked, a copy is still available in the [`luli/hedp`](https://github.com/luli/hedp/blob/master/hedp/math/abel.py#L107) repo, so feel free to either remove it or move it under `tools.symmery`, as you prefer. Thanks for fixing this issue, in general!

Reply:
Okay, I think that I will remove the function. Thanks!